### PR TITLE
added Cake.Plist by @reicheltp

### PR DESCRIPTION
**Pull request by ghuntley**

### CODE REVIEW

File: src/Cake.Web/App_Data/addins.xml [hunk 1]
```diff
@@ -364,4 +364,15 @@
...
   </Addin>
+  <Addin>
+    <Name>Cake.Plist</Name>
+    <NuGet Id="Cake.Plist">
+      <Filter>/**/Cake.Plist.dll</Filter>
+      <Filter>/**/Cake.Plist.xml</Filter>
+    </NuGet>
+    <Repository>https://github.com/reicheltp/Cake.Plist</Repository>
+    <Author>Paul Reichelt</Author>
+    <Description>Cake addin for deserializing, modifying and serializing Apple macOS/iOS xml plists directly from .NET source code.</Description>
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/addins.xml]: Shouldn't `macOS` be `OS X`?

Comment (ghuntley) [inline src/Cake.Web/App_Data/addins.xml]: OSX is now macOS :trollface: :-1: (god damn it apple)

http://www.theverge.com/2016/6/22/11997918/apple-macos-sierra-preview-siri-icloud-drive

Comment (gep13) [inline src/Cake.Web/App_Data/addins.xml]: @ghuntley until that happens, I think we should stick to OS X for now, and then we can do an update across the entire site when it ships.  Thoughts?

Comment (patriksvensson) [inline src/Cake.Web/App_Data/addins.xml]: @gep13 @ghuntley Yes, I think that is a good idea.

**Comment by ghuntley** [comment]
Changed to OSX instead of macOS.

**Comment by gep13** [comment]
@ghuntley thank you for making the update as requested.  This should be live on the site soon.

**Pull request merged**